src/clients/affiliate.ts

```diff
@@ -18,7 +18,7 @@ export class Affiliate extends JanusClient {
   }
 
   public registerAffiliate(
-    { name, id, salesChannelId, searchEndpoint }: AffiliateInput,
+    { name, id, salesChannelId, searchEndpoint, followUpEmail }: AffiliateInput,
```

Comment:
Bear in mind that this is a breaking change 💀  People using the client like it's today, if updating, will run into typing errors

Reply:
Should it be a major upgrade?

Reply:
yes, true 😢